code/modules/antag_contest/contest4_eridani_mercs.dm

```diff
@@ -0,0 +1,59 @@
+/hook/roundstart/proc/schedule_eridani_fax()
+	addtimer(CALLBACK(GLOBAL_PROC, /proc/send_eridani_fax), rand(1, 10) MINUTES)
+	return TRUE
+
+/proc/send_eridani_fax()
+	var/obj/machinery/photocopier/faxmachine/F = null
+
+	for (var/fx in allfaxes)
+		var/obj/machinery/photocopier/faxmachine/FX = fx
+		if (FX.department == "Head of Security's Office")
+			F = FX
+			break
+
+	if (!F)
+		return
+
+	var/list/objectives = list(
+		"A general alert exercise is to be carried out. Ensure the crew successfully carry out the correct procedure for each alert level for the shift." = 1,
+		"We believe [pick(station_departments - "Command")] department contains known regulation breakers. Investigate using any means available." = 1,
+		"A simulated [pick("hostile nation", "hostile corporation", "unknown life-form")] attack is to be carried out. Ensure the crew obey proper procedure and regulations." = 1,
+		"The [pick(station_departments - "Command")] department has been over funded. Aquire [rand(1000, 10000)] credits from their departmental account." = 1,
+		"Ensure the station is secure. There are no special objectives at this time." = 6
+	)
+	var/objective = pickweight(objectives)
+
+	var/fax_body = {"\[center\]\[logoeridani\]\[/center\]
+\[center\]\[b\]ERIDANI CORPORATE BOARD ENCODED TRANSMISSION\[/b\]\[/center\]
+\[center\]\[u\]\[small\]FOR USE BY ERIDANI CORPORATE ENFORCEMENT TROOPS ONLY\[/small\]\[/u\]\[/center\]
+
+The Corporate Board has decided to issue You the following objective for today's shift:
```

Comment:
`You` should not be capitalized here.

Reply:
huh, apparently it's an Estonian thing. https://english.stackexchange.com/questions/30185/you-versus-you-as-polite-form-of-writing